Mathlib/CategoryTheory/NatIso.lean

```diff
@@ -264,4 +264,23 @@ theorem isIso_map_iff {F₁ F₂ : C ⥤ D} (e : F₁ ≅ F₂) {X Y : C} (f : X
 
 end NatIso
 
+namespace Functor
+
+variable (F : C ⥤ D) (obj : C → D) (e : ∀ X, F.obj X ≅ obj X)
+
+/-- Constructor for a functor that is isomorphic to a given functor `F : C ⥤ D`,
+while being definitionally equal on objects to a given map `obj : C → D`
+such that for all `X : C`, we have an isomorphism `F.obj X ≅ obj X`. -/
+@[simps obj]
+def mkOfFunctorOfObj : C ⥤ D where
```

Comment:
We often use `copy` as the name for a decl that changes some data up to propeq, for better defeqs. Since you only change `obj`, maybe this could be called
```suggestion
def copyObj : C ⥤ D where
```

Reply:
Thanks very much for all these reviews! and for this suggestion, which I have just applied.